src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -236,6 +238,8 @@
     this.sourceEventOrdering = context.getSourceProperties() == null ? SourceProperties.Ordering.ORDERED :
       context.getSourceProperties().getOrdering();
     this.datasetName = datasetName;
+    this.retainStagingTable = Boolean.parseBoolean(context.getRuntimeArguments().getOrDefault(RETAIN_STAGING_TABLE,
```

Comment:
minor nit: no need to assign default. `null` value is treated as `false`. so could be simplified as 
```
this.retainStagingTable = Boolean.parseBoolean(context.getRuntimeArguments().get(RETAIN_STAGING_TABLE));
```

Reply:
done:)